packages/application-generic/src/usecases/get-novu-integration/get-novu-integration.usecase.ts

```diff
@@ -24,6 +29,7 @@ export class GetNovuIntegration {
     private integrationRepository: IntegrationRepository,
     private calculateLimitNovuIntegration: CalculateLimitNovuIntegration,
     private organizationRepository: OrganizationRepository,
+    @Inject(forwardRef(() => AnalyticsService))
```

Comment:
Forced to implement this. Need to investigate what's the circular dependency causing this.